packages/asset-uploader/src/delete.ts

```diff
@@ -3,24 +3,25 @@ import { deleteFromDb } from "./db";
 import { Asset } from "./types";
 import { deleteFromFs } from "./targets/fs/delete";
 import { deleteFromS3 } from "./targets/s3/delete";
+import { formatAsset } from "./utils/format-asset";
 
-export const deleteAsset = async ({
-  id,
-  name,
-}: {
-  id: string;
-  name: string;
-}): Promise<Asset> => {
-  const currentAsset = await prisma.asset.findUnique({
-    where: { id },
+export const deleteAssets = async (
+  ids: Array<string>
+): Promise<Array<Asset>> => {
+  const assets = await prisma.asset.findMany({
+    where: { OR: ids.map((id) => ({ id })) },
```

Comment:
I'd try `in` https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#in

Reply:
good hint, done